scss/_buttons.scss

```diff
@@ -77,6 +78,15 @@ fieldset:disabled a.btn {
 }
 
 // Custom Flo Primary
+.btn-primary{
+
+  &.disabled,
+  &:disabled {
+    background-color: $gray-400;
+    border-color: $gray-400;
+    opacity: 1;
+  }
+}
```

Comment:
I think we also need `fieldset:disabled` which bootstrap has.  https://github.com/twbs/bootstrap/blob/master/scss/_buttons.scss#L44

Reply:
Got it